scripts/build.py

```diff
@@ -712,6 +725,14 @@ def minify_func(source_path, target_path, file_hashes, filename):
     elif filename.endswith('.css') or filename.endswith('.js'):
         print 'Minifying %s' % source_path
         _minify(source_path, target_path)
+    elif filename.endswith('.ts'):
+        compile_typescript_files(source_path)
+        target_path = target_path.replace('.ts', '.js')
+        minified_target_path = target_path.replace('.js', '_min.js')
+        print 'Minifying %s' % target_path
+        _minify(target_path, minified_target_path)
+        os.remove(target_path)
+        os.rename(minified_target_path, target_path)
```

Comment:
Rather than doing this, save the intermediate thing as *.unmin.js and then keep the target as *.js, otherwise it's a bit confusing since the meaning of the path changes midway.

Reply:
Done.